pkg/local_object_storage/metabase/db_test.go

```diff
@@ -44,11 +45,11 @@ func testSelect(t *testing.T, db *meta.DB, cnr cid.ID, fs object.SearchFilters,
 }
 
 func newDB(t testing.TB, opts ...meta.Option) *meta.DB {
-	path := t.Name()
+	p := t.TempDir()
 
 	bdb := meta.New(
 		append([]meta.Option{
-			meta.WithPath(path),
+			meta.WithPath(path.Join(p, "meta.db")),
```

Comment:
Something different.

Reply:
you mean pick another name or what?

Reply:
Unrelated to the patch it's included into.